core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -59,6 +61,8 @@
     public static final String JOIN_MAX_BROADCAST_TABLE_SIZE = "join_max_broadcast_table_size";
     public static final String JOIN_MULTI_CLAUSE_INDEPENDENCE_FACTOR = "join_multi_clause_independence_factor";
     public static final String DISTRIBUTED_INDEX_JOIN = "distributed_index_join";
+    public static final String SKEWED_JOIN_METADATA = "skewed_join_metadata";
```

Comment:
Instead of `skewed_join_metadata`, this should be just `skewed_values` or something similar. There's nothing specific to joins in this option. In fact, it could be used for other operations such as aggregations, for instance.

Reply:
good point, i think aggregations and range joins could benefit.